conf/hosts.txt

```diff
@@ -1,3 +1,4 @@
 #This is the hosts file for Jasminegraph. All the host names on which Jasminegraph workers should get started needs
 #to be listed here
-localhost
\ No newline at end of file
+localhost
+ubuntu@18.219.24.222
```

Comment:
Shall we comment this line with # symbol? because this is just a value used in your example scenario.